dashkeys.js

```diff
@@ -1225,6 +1225,8 @@ var DashKeys = ("object" === typeof module && exports) || {};
 
   _DashKeys.utils = Utils;
 
+  //@ts-ignore
+  Window.DashKeys = _DashKeys;
```

Comment:
This is already exported at the top via `var DashKeys`. Does it _have_ to be exported _again_?

Reply:
Yes, it wouldn't work without that. Its not treated as being on Window/Global without it, so you can't do this to get the utils.

```js
import './node_modules/dashkeys/dashkeys.js';

/** @type {DashKeysTypes} */
export let DashKeys = window?.DashKeys || globalThis?.DashKeys
```